### DISCUSSION TITLE
refresh salt after global.sls edit?

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2339] [Unsupported Versions] [answered]
I edited the saltstack global.sls file to turn off Stenographer.  Do I need to refresh or reload anything for the change to take affect?  If so, what's the command?

**Comment by TOoSmOotH** [comment, marked as answer]
The sensor will check in within 15 minutes and will disable it. If you want to speed it up you can do sudo salt \*_sensor state.highstate from the manager.
